Tests/DatadogIntegrationTests/Scenarios/Logging/LoggingScenarioTests.swift

```diff
@@ -14,7 +14,7 @@ class LoggingScenarioTests: IntegrationTests, LoggingCommonAsserts {
 
         let app = ExampleApplication()
         app.launchWith(
-            testScenario: LoggingManualInstrumentationScenario.self,
+            testScenarioClassName: "LoggingManualInstrumentationScenario",
```

Comment:
I think it's fair to go with this more dynamic approach. As you said very well in the PR description, a mismatch should be caught early by running the tests.